curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/nest-an-anchor-element-within-a-paragraph.md

```diff
@@ -32,18 +32,18 @@ Next is the *anchor* element `<a>` (which requires a closing tag `</a>`):
 `target` is an anchor tag attribute that specifies where to open the link. The value `_blank` specifies to open the link in a new tab. The `href` is an anchor tag attribute that contains the URL address of the link:  
 
 ```html
-<a href="http://freecodecamp.org"> ... </a>
+<a href="http://freecodecamp.org" target="_blank"> ... </a>
 ```
 
 The text, `link to freecodecamp.org`, within the `a` element is called <dfn>anchor text</dfn>, and will display the link to click:
 
 ```html
-<a href=" ... ">link to freecodecamp.org</a>
+<a href=" ... " target="_blank">link to freecodecamp.org</a>
```

Comment:
```suggestion
<a href=" ... " target=" ... ">link to freecodecamp.org</a>
```

Yep I think I'm in favour of this syntax.

Reply:
Nicholas, I cannot find a reference to this syntax anywhere? Is it really worth it to save 1 character?

Reply:
I'm fine with it either way - it mirrors the `href` syntax right next to it.

If we want to leave the text there, we should put the `href` text there too.

Reply:
🤦 My bad... I did not realise this was in the example - I thought this was the _fix_